.github/workflows/webify_models_test.yml

```diff
@@ -0,0 +1,28 @@
+name: webify_models
+
+on: push
+# on:
+#   push:
+#     paths:
+#     - 'model_processing/crates/webify_models/**'
```

Comment:
This could be handy in the future, but I wanted to be sure it was running at this stage.